dashboard/components.py

```diff
@@ -2,7 +2,28 @@
 import dash
 import figures
 
-def add_figure_card(figure_id):
+def add_figure_card_full_row(figure_id):
```

Comment:
`def figure_card_full_row(figure_id):`
It would be better if you could remove the add from the name, in this method you do not add it to anywhere. You are getting the graph/div placeholder that later will be added.